gbe/forms/persona_form.py

```diff
@@ -47,7 +47,7 @@ def __init__(self, *args, **kwargs):
             )
 
     def save(self, commit=True):
-        performer = super(ModelForm, self).save(commit=False)
+        performer = super(PersonaForm, self).save(commit=False)
```

Comment:
These are not related, but in reviewing, I noticed that the code was not done properly.  There should be no effect.